support-services/src/main/scala/com/gu/support/redemption/corporate/DynamoTableAsync.scala

```diff
@@ -10,8 +10,8 @@ import software.amazon.awssdk.regions.Region
 import software.amazon.awssdk.services.dynamodb.DynamoDbAsyncClient
 import software.amazon.awssdk.services.dynamodb.model.{AttributeValue, GetItemRequest, UpdateItemRequest}
 
-import scala.collection.JavaConverters._
-import scala.compat.java8.FutureConverters._
+import scala.jdk.CollectionConverters._
+import scala.jdk.FutureConverters._
```

Comment:
this is how we convert from java `CompletableFuture` to scala `Future` in scala 2.13